Documentation/source/conf.py

```diff
@@ -237,11 +239,12 @@ def __getattr__(cls, name):
serList', 'NSLeafProxy', 'NSLeftArrowFunctionKey', 'NSLeftMarginDocumentAttribute', 'NSLeftMouseDown', 'NSLeftMouseDownMask', 'NSLeftMouseDragged', 'NSLeftMouseDraggedMask', 'NSLeftMouseUp', 'NSLeftMouseUpMask', 'NSLeftTabStopType', 'NSLeftTabsBezelBorder', 'NSLeftTextAlignment', 'NSLeftTextMovement', 'NSLegacyScrollerImp', 'NSLessThanComparison', 'NSLessThanOrEqualToComparison', 'NSLessThanOrEqualToPredicateOperatorType', 'NSLessThanPredicateOperatorType', 'NSLevelIndicator', 'NSLevelIndicatorCell', 'NSLibraryDirectory', 'NSLigatureAttributeName', 'NSLightGray', 'NSLighterFontAction', 'NSLikePredicateOperator', 'NSLikePredicateOperatorType', 'NSLineBorder', 'NSLineBreakByCharWrapping', 'NSLineBreakByClipping', 'NSLineBreakByTruncatingHead', 'NSLineBreakByTruncatingMiddle', 'NSLineBreakByTruncatingTail', 'NSLineBreakByWordWrapping', 'NSLineDoesntMove', 'NSLineFragmentRenderingContext', 'NSLineHeightFormatter', 'NSLineMovesDown', 'NSLineMovesLeft', 'NSLineMovesRight', 'NSLineMovesUp', 'NSLineSeparatorCharacter', 'NSLineSweepDown', 'NSLineSweepLeft', 'NSLineSweepRight', 'NSLineSweepUp', 'NSLineToBezierPathElement', 'NSLinearSlider', 'NSLinguisticTagger', 'NSLinkAttributeName', 'NSLinkCheckingResult', 'NSListModeMatrix', 'NSLiteralSearch', 'NSLoadedClasses', 'NSLocalDomainMask', 'NSLocalInputServer', 'NSLocalNotificationCenterType', 'NSLocalWindowWrappingRemoteWindow', 'NSLocale', 'NSLocaleAlternateQuotationBeginDelimiterKey', 'NSLocaleAlternateQuotationEndDelimiterKey', 'NSLocaleCalendar', 'NSLocaleCollationIdentifier', 'NSLocaleCollatorIdentifier', 'NSLocaleCountryCode', 'NSLocaleCurrencyCode', 'NSLocaleCurrencySymbol', 'NSLocaleDecimalSeparator', 'NSLocaleExemplarCharacterSet', 'NSLocaleGroupingSeparator', 'NSLocaleIdentifier', 'NSLocaleLanguageCode', 'NSLocaleLanguageDirectionBottomToTop', 'NSLocaleLanguageDirectionLeftToRight', 'NSLocaleLanguageDirectionRightToLeft', 'NSLocaleLanguageDirectionTopToBottom', 'NSLocaleLanguageDirectionUnknown', 'NSLocaleMeasurementSystem', 'NSLocaleQuotationBeginDelimiterKey', 'NSLocaleQuotationEndDelimiterKey', 'NSLocaleScriptCode', 'NSLocaleUsesMetricSystem', 'NSLocaleVariantCode', 'NSLocalizedDescriptionKey', 'NSLocalizedFailureReasonErrorKey', 'NSLocalizedKeyDictionaryBinding', 'NSLocalizedRecoveryOptionsErrorKey', 'NSLocalizedRecoverySuggestionErrorKey', 'NSLocalizedString', 'NSLocalizedStringFromTable', 'NSLocalizedStringFromTableInBundle', 'NSLocalizedStringWithDefaultValue', 'NSLocationInRange', 'NSLock', 'NSLog', 'NSLogPageSize', 'NSLogicalTest', 'NSMACHOperatingSystem', 'NSMacOSRomanStringEncoding', 'NSMacSimpleTextDocumentType', 'NSMachBootstrapServer', 'NSMachErrorDomain', 'NSMachPort', 'NSMachPortDeallocateNone', 'NSMachPortDeallocateReceiveRight', 'NSMachPortDeallocateSendRight', 'NSMacintoshInterfaceStyle', 'NSMagnifierWindowContentView', 'NSMainMenuWindowLevel', 'NSMakeCollectable', 'NSMakePoint', 'NSMakeRange', 'NSMakeRect', 'NSMakeSize', 'NSMallocException', 'NSManagedObject', 'NSManagedObjectCon
```

Comment:
newer versions of Sphinx have support for mocking modules. I’ve tried to implement it, but it was throwing some errors. leaving this comment here as a reminder.